src/directives/styleMap.ts

```diff
@@ -60,6 +60,13 @@ export const styleMap = directive((styleInfo: StyleInfo) => (part: Part) => {
       ((part.committer.element as HTMLElement).style as any)[name] = null;
```

Comment:
I think you need the same `--` check here for removal as well.

While you're at it, seems like the `part.committer.element as HTMLElement).style` expression can be pulled out earlier and used for both places.